src/main/java/uk/nhs/hee/tis/trainee/forms/repository/FormRPartARepository.java

```diff
@@ -33,4 +34,5 @@
   @Query(fields = "{traineeTisId:1, id:1, submissionDate:1, lifecycleState:1}")
   List<FormRPartA> findByTraineeTisId(String traineeTisId);
 
+  List<FormRPartA> findByLifecycleState(LifecycleState lifecycleState);
```

Comment:
I don't think we are using the list, using count / size of result.
Would it help to have the count query instead of retrieving the list?

Reply:
For PUT api, I need to verify if the draft ID from frontend is the same as what we get from the db. That's why I need the content instead of the count.

Reply:
Added the id projection and also modified it to `findByTraineeTisIdAndLifecycleState`